omegaconf/omegaconf.py

```diff
@@ -1037,6 +1030,38 @@ def _maybe_wrap(
         )
 
 
+def _resolver_wrapper(
+    config: BaseContainer,
+    parent: Container,
+    args: Tuple[Any, ...],
+    args_str: Tuple[str, ...],
+    name: str,
+    resolver: Resolver,
+    use_cache: bool,
+    pass_parent: bool,
+    pass_root: bool,
+) -> Any:
```

Comment:
The inlined version was significantly simpler because it inherits the context.
it's short enough that I prefer to keep it as an inline method.

What is the motivation for pulling it out of the register_new_resolver?

Reply:
Explained in https://github.com/omry/omegaconf/pull/635#discussion_r599854362 => resolving